src/components/popover-surface/popover-surface.scss

```diff
@@ -1,6 +1,10 @@
 @use '../../style/functions';
 @use '../../style/mixins';
 
+/**
+ * @prop --limel-popover-font-family: Font family override for the popover. Because the popover is rendered in a portal, it inherits its font from the body element. If the body's font is not what should be used for the popover, this property can be used to override the font.
```

Comment:
Since this var starts with `--limel-` prefix, it means this is an internal variable (which is perfect). But that also means we shouldn't add public docs for it, using `* @prop`. You could say this component itself is `private` so nobody sees this, but I'd still not do it. It's confusing.

Reply:
No, this isn't an internal variable. But it has to start with a prefix because it has to be a global variable, so we need the prefix to avoid conflicts with global variables that other libs might provide.

Why are we using `--limel-` to denote that a variable is internal?

Reply:
Yes, throughout Lime Elements, we have always used `--limel-` as a convention for our internal stuff, and when we need public variables, we always use the component's name first, like `--popover-xxx-yyy`

Reply:
I've removed this, including the variable itself.